mappings/net/minecraft/server/command/TpCommand.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_6170 net/minecraft/server/command/TpCommand
```

Comment:
TeleportCommand

Reply:
No, they are two different commands, `TeleportCommand` is already mapped.

Reply:
oh ok, i totally forgot they where two commands